src/test/java/nextstep/step3/domain/RacingCarsTest.java

```diff
@@ -3,15 +3,41 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import java.util.Arrays;
+import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class RacingCarsTest {
 	@Test
 	@DisplayName("자동차수만큼 레이싱카를 만든다.")
 	public void racingCarsTest() {
-		RacingCars racingCars = new RacingCars(Arrays.asList("adel","joy","ruby"));
+		RacingCars racingCars = new RacingCars("adel,joy,ruby");
 		assertThat(racingCars.getRacingCars().size()).isEqualTo(3);
```

Comment:
이렇게도 바꿔볼 수 있을 것 같아요!
```suggestion
		assertThat(racingCars.getRacingCars()).hasSize(3);
```

Reply:
9e60fc96e7bd1440d87f30538402471c0d6a7dc8 에서 반영함